### PR TITLE
[Indonesian Translation] Create README.id.md

### PR DESCRIPTION
**Pull request by ilhamm24**

### CODE REVIEW

**Review by esau-morais — CHANGES_REQUESTED** [review]
Thank you for contribute with Indonesian language!

_Note: You need to change the `translations/Translations.md` too. Follow [these steps](https://github.com/sabesansathananthan/React-GitHub-Resume/issues/15#issuecomment-704188333). Besides that, change the LICENSE link to `../LICENSE`_

**Comment by ilhamm24** [comment]
> Thank you for contribute with Indonesian language!
> 
> _Note: You need to change the `translations/Translations.md` too. Follow [these steps](https://github.com/sabesansathananthan/React-GitHub-Resume/issues/15#issuecomment-704188333). Besides that, change the LICENSE link to `../LICENSE`_

Thank you for your response, i just changed it.

File: translations/README.id.md [hunk 1]
```diff
@@ -83,4 +83,4 @@ cd React-GitHub-Resume
 
 ## 📄 License
```
Comment (sabesansathananthan) [inline translations/README.id.md:84]: is that same in Indonesia?? for varification

Comment (ilhamm24) [reply translations/README.id.md:84]: fixed it, thank you again for the correction

**Review by sabesansathananthan — CHANGES_REQUESTED** [review]
Note: You need to change the translations/Translations.md too.

**Comment by ilhamm24** [comment]
> Note: You need to change the translations/Translations.md too.

sorry, but i see in other translation they didnt changed that, so i just change the flag of indonesia

**Comment by esau-morais** [comment]
> > Note: You need to change the translations/Translations.md too.
> 
> sorry, but i see in other translation they didnt changed that, so i just change the flag of indonesia

All the PR's are being reviewed by me and I am trying to make sure all are following the [template](https://github.com/sabesansathananthan/React-GitHub-Resume/issues/15#issuecomment-704188333)

**Review by esau-morais — CHANGES_REQUESTED** [review]
Thank you for contribute with Indonesian language!!

_Note: Please, make sure to complete these tasks:_

- [ ] Add to the main `README.md` the country icon
- [x] Add to `translations/Translations.md` the language icon

_Note: To make sure you the icon is the right, follow [this template](https://github.com/sabesansathananthan/React-GitHub-Resume/issues/15#issuecomment-704188333)_

**Comment by sabesansathananthan** [comment]
could you please resolve conflicts. <3

**Comment by sabesansathananthan** [comment]
@esau-morais will you approve it?? @ilhamm24 please wait for the review...

**Comment by esau-morais** [comment]
> Thank you for contribute with Indonesian language!!
> 
> _Note: Please, make sure to complete these tasks:_
> 
> * [ ]  Add to the main `README.md` the country icon
> * [x]  Add to `translations/Translations.md` the language icon
> 
> _Note: To make sure you the icon is the right, follow [this template](https://github.com/sabesansathananthan/React-GitHub-Resume/issues/15#issuecomment-704188333)_

Just need to make the first change request

**Comment by sabesansathananthan** [comment]
@ilhamm24 update README.md in the root folder of the project.

**Review by esau-morais — APPROVED** [review]
Thank you for contribute with Indonesian language!!

_Note: Now it looks all good and following the [template](https://github.com/sabesansathananthan/React-GitHub-Resume/issues/15#issuecomment-704188333)

**Review by sabesansathananthan — APPROVED** [review]
Thank you for you PR

**Pull request merged**